clouddriver-titus/src/test/resources/mimicker-titus.yml

```diff
@@ -0,0 +1,2 @@
+mimicker:
+  titus:
```

Comment:
does this give you a place to define some inputs or configure the behavior for the instance of mimicker that you create?

Reply:
It gives you a place to define inputs. Example:

```yaml
mimicker:
  titus:
    cells:
    - cell1
    - cell2
    - cell3
```

```kotlin
dataContainer.list<String>("titus/cells") // returns full list of titus cells.
```